R/dissimilarity-based.R

```diff
@@ -8,10 +8,14 @@
 #' @description The default is to calculate Bray-Curtis dissimilarities after 
 #'  performing square-root transformations on the data. Send requests to add
 #'  additional metrics.
-#' @param tax_table The taxa table.
+#' @param tax_table$data_loaded The observations loaded into the taxa table.
```

Comment:
shouldn't include the $data_loaded here. This is the name of the parameter for this function. Not the name of the variable with the data